### ISSUE TITLE
paclair step type issues

### ISSUE THREAD

**Issue by dustinvanbuskirk** [codefresh-io/steps#4]
- No Clair image shown in UI. (Showing Gear Icon)
- No pull access to Docker registry.
```
(HTTP code 404) unexpected - pull access denied for codefreshplugins/paclair, repository does not exist or may require 'docker login'           
[SYSTEM]                                                                                                                                        
 Message             Failed to pull base image: codefreshplugins/paclair:latest       
```

**Comment by kostis-codefresh** [comment]
The docker image was fixed with commit 2794a438fe851069621313ce895518e384fd1b7d

**Comment by kostis-codefresh** [comment]
Fixed icon with commit 6b62139903e1530a00558e34d71fe995681671e8
![fixed-icon](https://user-images.githubusercontent.com/39800303/81702867-82374980-9474-11ea-8d63-c038e55b2f5b.png)
